resources/views/components/menu/management.blade.php

```diff
@@ -5,81 +5,66 @@
 
 /** @var User $user */
 $user = request()->user();
+
+$tools = $settings['tools'] ?? null;
 ?>
-<x-nav-dropdown dropdown-class="dropdown-menu-right" :title="__('Manage')">
+<x-nav-dropdown :class="$class ?? ''" dropdown-class="dropdown-menu-right" :title="__('Manage')">
     <x-slot name="trigger">
         <x-fas-toolbox/>
         <span class="ml-1 hidden sm:inline-block">{{ __('Manage') }}</span>
     </x-slot>
-    @can('develop')
-        @can('manage', App\Community\Models\TriggerTicket::class)
-            <x-dropdown-header>{{ __('Development') }}</x-dropdown-header>
-            {{--<x-dropdown-item :link="route('triggers.ticket.index')">{{ __res('ticket') }}</x-dropdown-item>--}}
-            <x-dropdown-item :link="url('ticketmanager.php')">{{ __res('ticket') }}</x-dropdown-item>
-            <x-dropdown-item :link="url('ticketmanager.php?f=1')">Most Reported Games</x-dropdown-item>
-            <x-dropdown-item :link="url('achievementinspector.php')">Achievement Inspector</x-dropdown-item>
-        @endcan
-        @can('manage', App\Community\Models\AchievementSetClaim::class)
-            <x-dropdown-item :link="url('expiringclaims.php')">Expiring Claims</x-dropdown-item>
-        @endcan
-        @can('manage', App\Platform\Models\GameHash::class)
-            <x-dropdown-item :link="url('latesthasheslinked.php')">Latest Linked Hashes</x-dropdown-item>
-        @endcan
-    @endif
-    @if($user->Permissions >= Permissions::Developer)
-        <x-dropdown-header>{{ __('Community') }}</x-dropdown-header>
-        @can('manage', App\Community\Models\News::class)
-            {{--<x-dropdown-item :link="route('news.index')">{{ __res('news') }}</x-dropdown-item>--}}
-            <x-dropdown-item :link="url('submitnews.php')">{{ __res('news') }}</x-dropdown-item>
-        @endcan
-        @can('manage', App\Community\Models\Forum::class)
-            {{--<x-dropdown-item :link="route('forum-topic.verify')">Forum Verification</x-dropdown-item>--}}
-            <x-dropdown-item :link="url('viewforum.php?f=0')">Forum Verification</x-dropdown-item>
-        @endcan
-        {{--@can('manage', App\Site\Models\Event::class)
-            <h6 class="dropdown-header">Events</h6>
-        @endcan--}}
-        {{--@can('manage', App\Platform\Models\IntegrationRelease::class)
-            <x-dropdown-header>Releases</x-dropdown-header>
-            @can('manage', App\Platform\Models\Emulator::class)
-                <x-dropdown-item :link="route('emulator.index')" :active="request()->routeIs('emulator*')">Emulators</x-dropdown-item>
-            @endcan
-            <x-dropdown-item :link="route('integration.release.index')" :active="request()->routeIs('integration.release*')">Integration</x-dropdown-item>
-        @endcan--}}
-        @if($user->Permissions >= Permissions::Moderator)
-            <x-dropdown-item :link="url('admin.php')">Admin Tools</x-dropdown-item>
+    <div class="md:flex">
```

Comment:
On sizes less than LG, the nav bar doesn't float (it scrolls with the page). Is that intentional?

Reply:
Yes, kinda. It's still the same behaviour as before and the easiest way to make sure dropdowns with long lists of items can be scrolled down fully without being cut off at the bottom on small screens. 
There are certainly more sophisticated solutions for that (make the dropdown fixed width, make itself scrollable, make menus scrolalble overlays to make proper use of small screens, etc) but that might be a good candidate for an improvement PR.